base_app/static/swagger.json

```diff
@@ -13,6 +13,40 @@
   "basePath" : "/api/v1",
   "schemes" : [ "http" ],
   "paths" : {
+    "/notify": {
+      "get" : {
+        "summary" : "sends an SMS",
+        "description" : "By providing the required options, this will send an SMS invitation to join a friend at a specified location.\n",
+        "produces" : [ "application/json" ],
+        "parameters" : [ {
+          "name" : "to",
+          "in" : "query",
+          "description" : "Provide an SMS-enabled phone number to receive a notification.",
+          "required" : true,
+          "type" : "integer"
+        }, {
+          "name" : "from",
+          "in" : "query",
+          "description" : "Provide a Twilio SMS-enabled phone number from which to send a notification.",
```

Comment:
Good call for differentiating the description and noting that this has to be a Twilio SMS-enabled number here.